yolact++_colab_demo.ipynb

```diff
@@ -0,0 +1,392 @@
+        "\n",
+        "> Colab author: [mrm8488](https://twitter.com/mrm8488)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "aVJsG_GUs1QZ",
+        "colab_type": "text"
+      },
+      "source": [
+        "# Install required packages"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "l1naq-OprvLQ",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "# Cython needs to be installed before pycocotools\n",
+        "!pip install cython\n",
+        "!pip install opencv-python pillow pycocotools matplotlib "
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "p9iG8ub89xRX",
+        "colab_type": "text"
+      },
+      "source": [
+        "# Clone the repository"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "rjkNTFlMtC2i",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 141
+        },
+        "outputId": "6eac3b02-3eaf-49a4-976c-3c2d43bd8a8c"
+      },
+      "source": [
+        "!git clone https://github.com/dbolya/yolact.git"
+      ],
+      "execution_count": 2,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "Cloning into 'yolact'...\n",
+            "remote: Enumerating objects: 20, done.\u001b[K\n",
+            "remote: Counting objects: 100% (20/20), done.\u001b[K\n",
+            "remote: Compressing objects: 100% (18/18), done.\u001b[K\n",
+            "remote: Total 2858 (delta 0), reused 7 (delta 0), pack-reused 2838\u001b[K\n",
+            "Receiving objects: 100% (2858/2858), 21.20 MiB | 15.44 MiB/s, done.\n",
+            "Resolving deltas: 100% (1942/1942), done.\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "kLMA2Ebxw5tT",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 35
+        },
+        "outputId": "99281b68-9b13-4ad8-ab8b-3ca220d01e93"
+      },
+      "source": [
+        "cd yolact"
+      ],
+      "execution_count": 3,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "/content/yolact\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "2tQjvVnCxQJ2",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!mkdir weights"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "i46hGwDh92VM",
+        "colab_type": "text"
+      },
+      "source": [
+        "# Download the model weights"
```

Comment:
You might add more details from the Readme, ie the different pretrained nets and their FPS/mAP as a documentation here. Ofc only one net is downloaded